### PR TITLE
Default language to configured value for login & setup

### PR DESCRIPTION
**Pull request by ylavoie**
Non logged in screens should use the configured default in ledgersmb.conf

### CODE REVIEW

File: lib/LedgerSMB/Template/UI.pm [hunk 1]
```diff
@@ -32,7 +32,7 @@ our $singleton;
 our @pre_render_cbs = (
     sub {
         my ($request, $template, $vars, $cvars) = @_;
-        $vars->{USER} = $request->{_user};
+        $vars->{USER} = $request->{_user} // { language => LedgerSMB::Sysconfig::language() };
```
Comment (ehuelsmann) [inline lib/LedgerSMB/Template/UI.pm:35]: Why do we need this one? You handled setup and login in the other two changes...

File: UI/setup/edit_user.html [hunk 2]
```diff
@@ -50,9 +50,12 @@
                 <td><label for="password">[% text('Password') %]</label>
                 </td>
                 <td>
-                    <input type="password"
-                           id="password" name="password" value=""
-                           autocomplete="off" />
+                  [% PROCESS input element_data = {
```
Comment (ehuelsmann) [inline UI/setup/edit_user.html:53]: Hmm. why did you include this commit in the PR? It's already on master, no?

Comment (ylavoie) [reply UI/setup/edit_user.html]: Rebase to remove it

File: lib/LedgerSMB/Template/UI.pm (hunk 1 above)
Comment (ylavoie) [reply lib/LedgerSMB/Template/UI.pm:35]: _user is only set for a logged in user

Comment (ehuelsmann) [reply lib/LedgerSMB/Template/UI.pm:35]: True, but in which situations does that happen, now that you override the content of `_user` in login and setup handling?

Comment (ylavoie) [reply lib/LedgerSMB/Template/UI.pm:35]: Every time a user is not logged. I only override `{user}` default language in login and setup, not `{_user}`.

Comment (ehuelsmann) [reply lib/LedgerSMB/Template/UI.pm:35]: Are you telling me we have both `user` *and* `_user` keys in `$request`? can we eliminate `user` in favor of having only `_user`?

Comment (ylavoie) [reply lib/LedgerSMB/Template/UI.pm]: Fixed

**Pull request merged**